Document/CategoryManager.php

```diff
@@ -22,7 +22,7 @@ class CategoryManager extends BaseDocumentManager implements CategoryManagerInte
     /**
      * {@inheritdoc}
      */
-    public function getPager(array $criteria, $page, $maxPerPage = 10)
+    public function getPager(array $criteria, $page, $limit = 10, array $sort = array())
```

Comment:
@Soullivaneuh To be more concise, versions are :
```
classification-bundle: 2.2.x-dev (~2.2@dev in composer.json)
code-bundle: 2.3.9 (not specified in composer.json)
```

This patch have been already merged in master (see https://github.com/sonata-project/SonataClassificationBundle/commit/da9ab9c463ee4a4979d0946f47980421421136a3 and https://github.com/sonata-project/SonataClassificationBundle/commit/890ac2114618ee5824f2ef0bc7644988ac22dab6)